packages/react/src/main-nav/MainNavContainer.tsx

```diff
@@ -46,10 +45,10 @@ export function MainNavContainer({
 			{!focusMode && (
 				<Flex
 					css={{ margin: '0 auto' }}
+					flexWrap={{ xs: 'wrap', lg: 'wrap-reverse' }}
```

Comment:
What do you think about adding a comment for why `'wrap-reverse'` was added? I just fear it's value will be forgotten in time and removed.

Reply:
Yep, can do that

Reply:
Comment added